src/components/event/index.js

```diff
@@ -24,7 +24,7 @@ class Event extends React.Component {
     return <div className="uk-container uk-margin-large-top">
       <div uk-grid="true" className="uk-grid-large uk-child-width-expand@s">
         <div className={'uk-width-1-2@s'}>
-          <img src="/fest-logo.png" alt=""></img>
+          <img src={event.img ? event.img: "/fest-logo.png"} alt=""></img>
```

Comment:
Add some css to make the image large.

Reply:
It is already large enough, please check locally first.